src/Models/QueueMonitor.php

```diff
@@ -94,4 +95,14 @@ public function hasSucceeded(): bool
 
         return ! $this->hasFailed();
     }
+
+    /**
+     * Get the prunable model query.
+     *
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function prunable()
+    {
```

Comment:
it could be nice to make it activable (see config option) : 
```suggestion
    {
              if ( config('filament-jobs-monitor.pruning') ) {
                 ....
              }
              
              return false;
    }
```